examples/test.py

```diff
@@ -5,6 +5,9 @@
 # country boundaries, filling continents and drawing
 # parallels/meridians
 
+import matplotlib
+matplotlib.rcParams['figure.max_open_warning'] = 0
```

Comment:
This should have a test that the parameter exists first before trying to set it. This would error out for older versions of matplotlib.